smack-bosh/build.gradle

```diff
@@ -4,5 +4,5 @@ This API is considered beta quality."""
 
 dependencies {
 	compile project(':smack-core')
-	compile 'org.igniterealtime.jbosh:jbosh:[0.9,0.10)'
+	compile 'org.igniterealtime.jbosh:jbosh:[0.10,0.11)'
```

Comment:
I was actually planning to add support for HTTP headers to jbosh 0.9.1, so the version range would become `[0.9.1,0.10)`. (And that are the things I wanted to discuss with you in IRC ;))